src/coreclr/jit/codegenxarch.cpp

```diff
@@ -2216,36 +2216,32 @@ void CodeGen::genAllocLclFrame(unsigned frameSize, regNumber initReg, bool* pIni
 //
 // Arguments:
 //    spDelta                 - the value to add to SP. Must be negative or zero.
-//    regTmp                  - x86 only: an available temporary register. If not REG_NA, hide the SP
-//                              adjustment from the emitter, using this register.
+//    canTrack                - x86 only: whether or not to track the SP adjustment
```

Comment:
`canTrack` seems like an odd name. Makes it sound optional. What not something more like `trackSpAdjustments`?

Reply:
Makes sense, though `trackSpAdjustments` is a bit longer, but clearer.